ibm_catalog.json

```diff
@@ -1,79 +1,134 @@
 {
-    "products": [
-      {
-        "name": "deploy-arch-ibm-slz-mas",
-        "label": "Maximo Application Suite",
-        "product_kind": "solution",
-        "tags": [
-          "ibm_created",
-          "target_terraform",
-          "terraform",
-          "reference_architecture",
-          "solution"
-        ],
-        "keywords": [
-          "vpc",
-          "slz",
-          "IaC",
-          "infrastructure as code",
-          "terraform",
-          "solution",
-          "mas",
-          "maximo",
-          "Maximo Application Suite"
-        ],
-        "short_description": "Installs IBM Maximo Application Suite on IBM Cloud landing zone",
-        "long_description": "Installs IBM Maximo Application Suite on IBM Cloud landing zone",
-        "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-mas/blob/main/README.md",
-        "offering_icon_url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-mas/main/images/ibm-icon.svg",
-        "provider_name": "IBM",
-        "features": [
-          {
-            "description": "Supports installing the Maximo Application Suite on existing IBM landing zone cluster.",
-            "title": "Installs the Maximo Application Suite on existing IBM landing zone cluster."
-          }
-        ],
-        "flavors": [
-          {
-            "label": "Existing Cluster",
-            "name": "ExistingCluster",
-            "install_type": "extension",
-            "working_directory": "solutions/existing-cluster",
-            "dependencies": [
-              {
-                "flavors": [
-                  "standard"
-                ],
-                "id": "95fccffc-ae3b-42df-b6d9-80be5914d852-global",
-                "name": "deploy-arch-ibm-slz-ocp",
-                "version": ">=v3.0.0"
-              }
-            ],
-            "architecture": {
-              "descriptions": "",
-              "features": [
-                {
-                  "title": "Works with the landing zone",
-                  "description": "Yes"
-                },
-                {
-                  "title": "Supports installing the Maximo Application Suite on existing IBM landing zone cluster.",
-                  "description": "Yes"
-                }
-              ],
-              "diagrams": [
-                {
-                  "diagram": {
-                    "caption": "Maximo Application Suite on existing IBM landing zone cluster.",
-                    "url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-mas/main/images/MAS_DA_Architecture_Diagram.drawio.svg",
-                    "type": "image/svg+xml"
-                  },
-                  "description": "Maximo Application Suite on existing IBM landing zone cluster."
-                }
-              ]
-            }
-          }
-        ]
-      }
-    ]
-  }
+	"products": [
+		{
```

Comment:
json generally uses 2 space indentation, and thats what is in main branch. It seems your IDE is adding full tabs for indentation, making it impossible for me to see the diff of the changes you are making to this file. Can you go back to 2 space indentation?